doc/asab/pubsub.rst

```diff
@@ -206,13 +206,19 @@ This message is emitted when application receives UNIX signal ``SIGHUP`` or equi
 
 .. option:: Application.housekeeping!
 
-This message is published when application is on the time for housekeeping. The time for housekeeping is set to 03:00 AM UTC. It can be changed in the configuration file:
+This message is published when application is on the time for housekeeping. The time for housekeeping is set to 03:00 AM UTC by default.
+
+The app listens every ten minutes to see if it's time for housekeeping. If the UTC time reaches the value for housekeeping, the app will publish it and set the time for the next housekeeping for the next day at the same time.
+There is also a time limit, which is set to 05:00 AM UTC by default. If the computer is in a sleep state, housekeeping will not be performed. Then, when the computer is reawakened again, it will check if it has exceeded the time limit. If not, then housekeeping will be published. If it has exceeded it, it simply informs the user and sets the housekeeping time for the next day.
```

Comment:
i would add:
"Note that this only limits the time when the housekeeping can start. If the housekeeping event triggers a procedure that takes a long time to finish, it will not be terminated when the time limit is reached."